cyral/resource_cyral_repository_local_account.go

```diff
@@ -253,6 +259,24 @@ type RepositoryLocalAccountResource struct {
 func (repoAccount RepositoryLocalAccountResource) WriteToSchema(d *schema.ResourceData) error {
 	log.Printf("[DEBUG] RepositoryLocalAccountResource - WriteToSchema START")
 
+	// Optional `config` field
+	if repoAccount.Config != nil {
+		config := repoAccount.Config
+		configMap := make(map[string]interface{})
+		configMap["auto_approve_access"] = config.AutoApproveAccess
+		configMap["max_auto_approve_duration"] = config.MaxAutoApproveDuration
+		d.Set("config", []interface{}{configMap})
+	}
+
+	configList := d.Get("config").(*schema.Set).List()
+	if len(configList) > 0 {
+		configMap := configList[0].(map[string]interface{})
+		repoAccount.Config = &RepoAccountConfig{
+			AutoApproveAccess:      configMap["auto_approve_access"].(bool),
+			MaxAutoApproveDuration: configMap["max_auto_approve_duration"].(string),
+		}
+	}
```

Comment:
Why do we need this part of the code in the WriteToSchema? I believe this can be removed.

Reply:
Haha I don't know how this got here! Removed it in latest commit, and retested code.